griffon/cli.py

```diff
@@ -147,7 +147,9 @@ def cli(
     ctx.obj["SHOW_PURL"] = False
     ctx.obj["SHOW_UPSTREAM"] = False
     ctx.obj["FORMAT"] = format
-    ctx.obj["VERBOSE"] = verbose
+    ctx.obj["VERBOSE"] = 0
```

Comment:
I am confused, shouldn't the `get_config_option("default", "verbosity", 0)` ensure that there is always default value. How is this any different?

Reply:
this is an old c habit ... agreed we can make function setting defaults nicer but a param with value of 0 will be False ... hence why I just marked it out explicitly